libs/langgraph/langgraph/pregel/__init__.py

```diff
@@ -984,12 +1019,7 @@ def put_checkpoint(metadata: CheckpointMetadata) -> Iterator[Any]:
                     # increment start to 0
                     start += 1
                 else:
-                    # no input is taken as signal to proceed past previous interrupt
-                    checkpoint = copy_checkpoint(checkpoint)
-                    for k in channels:
-                        if k in checkpoint["channel_versions"]:
-                            version = checkpoint["channel_versions"][k]
-                            checkpoint["versions_seen"][INTERRUPT][k] = version
+                    raise ValueError("Received no input")
```

Comment:
@nfcampos technically we shouldn't even get here, not sure what's the cleanest thing to do here

Reply:
Let's create a new exception class for this and use run line 1061 too. eg. `EmptyInputError`